### PR TITLE
adding an install task that will publish to mavenLocal. Useful for de…

### PR DESCRIPTION
**Pull request by rcordeau**
…veloping locally.

---
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_schema/85)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -85,6 +85,9 @@ publishing {
     }
 }
 
+//Install task for installing (publishing) to local .m2
+task install(dependsOn: publishToMavenLocal)
```
Comment (rcordeau) [inline build.gradle]: Yeah, we could not have this and just use "./gradlew publishToMavenLocal", but "install" is less typing.

Comment (malaffoon) [inline build.gradle]: 👍

Comment (agorina) [inline build.gradle]: yeah!!!! no more submodules?

Comment (rcordeau) [inline build.gradle]: Nope.

Comment (chdietz) [inline build.gradle]: nice!

**Pull request merged**